intro-HPC/ch_openFOAM.tex

```diff
@@ -0,0 +1,319 @@
+
+There are currently three different sets of versions of OpenFOAM available, each with its own versioning scheme:
+\begin{itemize}
+    \item OpenFOAM versions released via \url{http://openfoam.com}: \lstinline|v3.0+|, \lstinline|v1706|
+    \begin{itemize}
+        \item see also \url{http://openfoam.com/history/}
+    \end{itemize}
+    \item OpenFOAM versions released via \url{https://openfoam.org}: \lstinline|v4.1|, \lstinline|v5.0|
+    \begin{itemize}
+        \item see also \url{https://openfoam.org/download/history/}
+    \end{itemize}
+    \item OpenFOAM versions released via \url{http://wikki.gridcore.se/foam-extend}: \lstinline|v3.1|
+\end{itemize}
+
+Make sure you know which flavor of OpenFOAM you want to use, since there are important differences between
+the different versions w.r.t. features.
+If the OpenFOAM version you need is not available yet, see \autoref{sec:software-installation}.
+
+\section{Documentation \& training material}
+\label{sec:best-practices-openfoam-documentation}
+
+The best practices outlined here focus specifically on the use of OpenFOAM on the VSC \hpc infrastructure.
+As such, they are intented to augment the existing OpenFOAM documentation rather than replace it.
+For more general information on using OpenFOAM, please refer to:
+\begin{itemize}
+\item OpenFOAM websites:
+\begin{itemize}
+    \item \url{https://openfoam.com}
+    \item \url{https://openfoam.org}
+    \item \url{http://wikki.gridcore.se/foam-extend}
+\end{itemize}
+\item OpenFOAM user guides:
+    \begin{itemize}
+    \item \url{https://www.openfoam.com/documentation/user-guide}
+    \item \url{https://cfd.direct/openfoam/user-guide/}
+    \end{itemize}
+\item OpenFOAM C++ source code guide: \url{https://cpp.openfoam.org}
+\item tutorials: \url{https://wiki.openfoam.com/Tutorials}
+\item recordings of "\emph{Introduction to OpenFOAM}" training session at UGent (May 2016):\\
+      \small{\url{https://www.youtube.com/playlist?list=PLqxhJj6bcnY9RoIgzeF6xDh5L9bbeK3BL}}
+\end{itemize}
+Other useful OpenFOAM documentation:
+\begin{itemize}
+\item {\small\url{https://github.com/ParticulateFlow/OSCCAR-doc/blob/master/openFoamUserManual_PFM.pdf}}
+\item \url{http://www.dicat.unige.it/guerrero/openfoam.html}
+\end{itemize}
+
+\section{Preparing the environment}
+\label{sec:best-practices-openfoam-environment}
+
+To prepare the environment of your shell session or job for using OpenFOAM,
+there are a couple of things to take into account.
+
+\subsection{Picking and loading an \texttt{OpenFOAM} module}
+First of all, you need to pick and load one of the available \lstinline|OpenFOAM| modules.
+To get an overview of the available modules, run `\lstinline|module avail OpenFOAM|'. For example:
+
+\begin{prompt}
+%\shellcmd{module avail OpenFOAM}%
+
+------------------ /apps/gent/CO7/sandybridge/modules/all ------------------
+   OpenFOAM/2.4.0-intel-2017a     OpenFOAM/3.0.1-intel-2016b
+   OpenFOAM/4.0-intel-2016b       OpenFOAM/4.1-intel-2017a
```

Comment:
Please update to mention the more recent `OpenFOAM` module versions while you're at it.